test/setup-fedora-govm.sh

```diff
@@ -52,17 +52,17 @@ gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cl
 EOF
 
     # For the sake of reproducibility, use fixed versions.
-    # List generated with:
-    # for v in 1.13 1.14 1.15 1.16 1.17 1.18; do for i in kubelet kubeadm kubectl; do echo "$i-$(sudo dnf --showduplicates list kubelet | grep " $v"  | sed -e 's/.* \([0-9]*\.[0-9]*\.[0-9]*[^ ]*\).*/\1/' | sort -u -n -k 2.6 | tail -n 1)"; done; done
-    # Note that -n and -k 2.6 are needed to handle minor versions > 9, and -k 2.6 works only if k8s major versions have form X.XX
+    # List of latest packages can be generated on Fedora with:
+    # for v in $(seq 13 20); do for i in kubelet kubeadm kubectl; do echo "$i-$(sudo dnf --showduplicates list kubelet | grep " 1.$v"  | sed -e 's/.* \([0-9]*\.[0-9]*\.[0-9]*[^ ]*\).*/\1/' | sort -u -n -k 1.6 | tail -n 1)"; done; done
+    # Note that -n and -k 1.6 are needed to handle minor versions > 9, and -k 1.6 works only if k8s major versions have form X.XX
```

Comment:
What are the fields that we are sorting on?

Reply:
good question! Now that I look at that again, I see we can (?) leave sorting out, if we trust that dnf keeps showing entries in sorted order already. This is what we get before "sort" now:
kubectl-1.18.0-0
1.18.1-0
1.18.2-0
1.18.3-0
1.18.4-0
1.18.4-1
1.18.5-0
1.18.6-0
1.18.8-0
1.18.9-0
1.18.10-0
1.18.12-0
1.18.13-0
1.18.14-0
1.18.15-0
1.18.16-0
---------------
so if we just take last line, it will be what we need.

The history of that example cmd is, before "sort -n -u -k 2.6" that I introduced, there was sort -u, which did not work for minor versions >9 as it always showed 1.18.9 after 1.8.10. I tried to improve it by telling that we need numeric sort, but that had to be applied only at certain sub-field (position 6) to work correctly. It solved the "9 before 1" issue, and it works as in current PR.
But indeed, seems we can simplify by leaving out sort as dnf shows entries in correct versioning order already.
The line without "sort" part currently produces same result as with "sort" part.